### PR TITLE
Make the webapp responsive and mobile-friendly #3

### LINKED ISSUE #3: Make the webapp responsive and mobile-friendly
* Bootstrap enhancements
* UI Improvement
* Content Placement

### PR DESCRIPTION
**Pull request by anand-nakat**
Fixes #3

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -167,6 +167,11 @@ function initMap() {
     </div>
   </section><!-- End Hero -->
 
+  <!-- Welcome Tooltip -->
+  <div id="welcome-tooltip">
+      Welcome Admin! <!--Username goes here -->
```
Comment (Chintan2108) [inline index.php]: @anand-nakat Please make this dynamic, so that the name of the signed in user is displayed.
You can use the php username variable here.

**Review by Chintan2108 — CHANGES_REQUESTED** [review]
@anand-nakat Please make the said changes

**Comment by Chintan2108** [comment]
@anand-nakat Also integrate the code changes for the list overflow fixes in this PR itself

File: css/main.css [hunk 2]
```diff
@@ -13,13 +13,17 @@
...
+  padding: 0rem;
   -webkit-box-sizing: border-box;
           box-sizing: border-box;
   color: #f1f2f6;
 }
 
+html{
+  font-size: 62.5%; /*1rem=10px*/
+}
+
```
Comment (anand-nakat) [inline css/main.css:26]: I'd also missed rem conversion for login page.. So adding it now in this commit

File: index.php [hunk 3]
```diff
@@ -104,7 +104,7 @@ function initMap() {
     </div>
     <!-- MENU -->
     <div class="nav-username">
```
Comment (anand-nakat) [inline index.php:106]: Dynamic Username Added

File: index.php [hunk 4]
```diff
@@ -126,7 +126,7 @@ function initMap() {
         <!-- Menu Body -->
         <div class="menu-body">
             <div class="menu-username">
-              <h2>Welcome Admin!</h2>
+              <h2>Welcome <?php echo($_SESSION['uname'])?>!</h2>
```
Comment (anand-nakat) [inline index.php:129]: Dynamic Name added

**Review by Chintan2108 — APPROVED** [review]
@anand-nakat Changes approved for merging

**Comment by Chintan2108** [comment]
give @anand-nakat 50 points

**Comment by Chintan2108** [comment]
give @anand-nakat 25 points

**Comment by Chintan2108** [comment]
@anand-nakat I've merged your PR, you can go ahead and delete the branch.

**Pull request merged**